frontend/src/components/Profile/ProfileLoaded.js

```diff
@@ -0,0 +1,30 @@
+import {Typography} from "antd";
+import ProfileStats from "./ProfileStats";
+import ProfileAbout from "./ProfileAbout";
+import ProfileSignature from "./ProfileSignature";
+
+
+
+const ProfileLoaded = ({profile}) => {
+
+    const { Title } = Typography
+    let stats = {
+        createdAt: profile.createdAt,
+        postAmount: profile.postAmount,
+        threadAmount: profile.threadAmount,
+        roles: profile.roles,
+        visits: profile.visits }
+
+    console.log(stats)
+
+    return (
+        <>
+            <Title level={4} style={{textAlign: 'center'}}>{profile['displayName']}'s profile</Title>
+            <ProfileStats stats = {stats} />
+            <ProfileAbout username={profile.username} about = {profile.about} />
```

Comment:
I think I'm going gaga because I didn't think spaces were allowed around the equation mark (this and previous line)